### PR TITLE
Task #198149- Reset password

### PR DESCRIPTION
**Pull request by Ekta29yadav**

### CODE REVIEW

File: modules/auth/src/pages/ForgetPassword.js [hunk 1]
```diff
@@ -332,7 +333,7 @@ export default function ForgetPassword() {
             )}
           </VStack>
           {OtpValue ? (
-            <Button
+            <FrontEndTypo.Primarybutton
               flex={1}
               variant={"primary"}
```
Comment (sagarkoshti1990) [inline modules/auth/src/pages/ForgetPassword.js:338]: remove variant from everywhere.

File: modules/auth/src/pages/Login.js [hunk 2]
```diff
@@ -210,18 +211,18 @@ export default function Login() {
...
-          </Button>
+          </FrontEndTypo.Primarybutton>
           <BodyMedium color="primary.500" textAlign="center">
-            <H2
+            <FrontEndTypo.H2
               onPress={() => {
                 navigate("/reset-password");
               }}
             >
               {" "}
```
Comment (sagarkoshti1990) [inline modules/auth/src/pages/Login.js:223]: remove `{" "}`

File: modules/auth/src/pages/ForgetPassword.js (hunk 1 above)
Comment (Ekta29yadav) [reply modules/auth/src/pages/ForgetPassword.js]: done

File: modules/auth/src/pages/Login.js (hunk 2 above)
Comment (Ekta29yadav) [reply modules/auth/src/pages/Login.js]: done

**Pull request merged**